src/Automation/Data/ScanOptions.cs

```diff
@@ -14,6 +14,11 @@ public class ScanOptions
         /// </summary>
         public string ScanId { get; }
 
+        /// <summary>
+        /// The window handle for the root of the UIA subtree to scan.
+        /// </summary>
+        public System.IntPtr WindowHandle { get; set; } = System.IntPtr.Zero;
```

Comment:
We've tried to expose every `ScanOptions` parameter to the CLI. Is there a similar plan to add that ability for the window handle? It's just a number, so plumbing it through should be straightforward.

Reply:
Added in latest iteration.